pom.xml

```diff
@@ -152,11 +167,6 @@
             <artifactId>powsybl-config-classic</artifactId>
             <scope>runtime</scope>
         </dependency>
-        <dependency>
```

Comment:
Need to restore after at next powsybl-ws-dependencies upgrade ?

Reply:
yes but we can keep the one above without the version number

Reply:
It just gathers the network store dependencies